src/ui/libs/DatalensChartkit/modules/menu/menu.ts

```diff
@@ -58,9 +58,6 @@ export const getChartkitMenuByType = (props?: GetChartkitMenuByType) => {
             },
             [MenuItemsIds.EDIT]: {
                 title: i18n('dash.chartkit-menu.view', 'button_edit'),
-                action: ({propsData: {id, params}}) => {
```

Comment:
Why don't we need it?

Reply:
It is override default behaviour, but probably we can delete it later